src/tools/generateConfigSectionForReadme.ts

```diff
@@ -0,0 +1,15 @@
+import { readFileSync } from "fs";
+import { join } from "path";
+import { cwd } from "process";
+
+const packageJsonPath: string = join(cwd(), "package.json");
+const packageJson = JSON.parse(readFileSync(packageJsonPath));
```

Comment:
Change `readFileSync(packageJsonPath)` to `readFileSync(packageJsonPath, {encoding: 'utf8'})` to fix travis